resources/user-agents/browsers/firefox/firefox-generic.json

```diff
@@ -154,43 +154,6 @@
           ]
         },
         {
-          "match": "Mozilla/5.0 (#PLATFORM#) Gecko*/*",
```

Comment:
This was discussed some in #961. These patterns would catch a lot of the patterns that are now being caught by the Mozilla browser rules. IMO, the agents they were catching weren't firefox, but just a Gecko based browser that didn't pass any other rules.